packages/grid/src/HeaderWrapper.tsx

```diff
@@ -146,6 +154,14 @@ export const HeaderWrapper: React.FC<IHeaderWrapper> = ({
           theme={ theme }
         >
           <span>{ movingColumnDataRef.current?.headerName }</span>
```

Comment:
This block repeats logic from HeaderCell component.
It is better create HeaderCellContent and put this logic inside it.